sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -977,3 +974,141 @@ def pct_change_overload(df, periods=1, fill_method='pad', limit=None, freq=None)
     ser_par = {'periods': 'periods', 'fill_method': 'fill_method', 'limit': 'limit', 'freq': 'freq'}
 
     return sdc_pandas_dataframe_apply_columns(df, name, params, ser_par)
+
+
+def sdc_pandas_dataframe_isin_iter_codegen(df, values, func_name, ser_param):
+    all_params = ['df', 'values']
+
+    df_func_name = f'_df_{func_name}_impl'
+    func_text, global_vars = _dataframe_apply_columns_codegen(func_name, all_params, ser_param, df.columns)
+
+    loc_vars = {}
+    exec(func_text, global_vars, loc_vars)
+    _apply_impl = loc_vars[df_func_name]
+    return _apply_impl
+
+
+def sdc_pandas_dataframe_isin_ser_codegen(func_name, all_params, columns):
+    result_name = []
+    joined = ', '.join(all_params)
+    func_lines = [f'def _df_{func_name}_impl({joined}):']
+    for i, c in enumerate(columns):
+        result_c = f'result_{c}'
+        func_lines += [
+            f'  series_{c} = pandas.Series(get_dataframe_data({all_params[0]}, {i}))',
+            f'  result = []',
+            f'  for i in range(len(series_{c}._data)):',
+            f'    if series_{c}._data[i] == values._data[i]:',
```

Comment:
Why are you comparing elements of the same indexes from series and values, is this correct?
Most likely the check should be 
`"if series_{c}._data[i] in values._data:'"`